### ISSUE TITLE
Failure to load static public/index.html when defined in package.json

### ISSUE THREAD

**Issue by kristoferjoseph** [architect/architect#1031]
## tl;dr
When a user defines their architect configuration in `package.json` the default static behavior breaks.

## Steps to repro
1. Add `architect` section to package.json only declaring `app` and `static` pragmas. 
```
{
  "architect": {
    "app": "issue",
    "static": {
      "fingerprint": true
    }
  },
  "scripts": {
    "start": "npx sandbox"
  },
  "devDependencies": {
    "@architect/architect": "^8.3.7-RC.1"
  }
}

```
2. add a `public/` directory
3. Add an `index.html` to `public/`
4. run `npx start`

## Expected
- Server should be started at `localhost:3333`
- navigating to `localhost:3333` should load `public/index.html`

## Actual
- Server is not started
- `public/index.html` is not served

## Screenshots
<img width="714" alt="Screen Shot 2020-12-14 at 10 04 21 AM" src="https://user-images.githubusercontent.com/76308/102117985-ebf54880-3df3-11eb-8c81-29272ce8539f.png">

**Comment by ryanblock** [comment]
Looks like this was introduced in the 3.2 series somewhere, thanks for the heads up!

**Comment by ryanblock** [comment]
Will be relased in 8.4, fixed here:
- https://github.com/architect/sandbox/pull/532/commits/39b4a7ec047b5427bb1486dda816381ee6439003
- https://github.com/architect/inventory/pull/10/commits/388a330b54e2901ddc0772f2a66c3da6cf0a2550
- https://github.com/architect/inventory/pull/10/commits/be4096bd553cddb141dc8f7a8e4b336b03b477b1
